languages/de.yml

```
@@ -0,0 +1,17 @@
+appendix: Anhänge
+references: Referenzen
+further-reading: Weitere Literatur
+bibliography-subsections:
+  standards: Normen
+  documents: ISTQB® Dokumente
+  books: Bücher
+  articles: Artikel
+reference-labels:
+  section: Kapitel
+  figure: Abbildung
+  table: Tabelle
+page: [Seite, von]
+provided-by: [Zur Verfügung gestellt von, Zur Verfügung gestellt von]
+istqb: International Software Testing Qualifications Board
+list-delimiters: [und, ',', ', und']
```

I don't think German uses [the Oxford comma][1], i.e. it should always be just `und`, even in lists of three or more items:

```suggestion
list-delimiters: [und, ',', und]
```

This is similar to `cs.yml` and `sk.yml`, where `list-delimiters[0] == list-delimiters[2]` also.

 [1]: https://www.oxford-royale.com/articles/oxford-comma/

---

@jsabak can you please comment on this one?